src/solver_interface.jl

```diff
@@ -142,6 +142,14 @@ Get the number of linear constraints in the model `m`.
 function get_number_linear_constraints end
 @deprecate lqs_getnumrows get_number_linear_constraints
 
+"""
+    get_last_linear_constraint_index(m)::Int
+
+Get the index of the last linear constraints added to the model `m`.
+"""
+function get_last_linear_constraint_index end
+get_last_linear_constraint_index(m) = get_number_linear_constraints(m)
```

Comment:
Ditto with comments. Something along the lines of users should only implement this if the indexing is different